blueflood-core/src/integration-test/java/com/rackspacecloud/blueflood/io/TestIO.java

```diff
@@ -0,0 +1,24 @@
+package com.rackspacecloud.blueflood.io;
+
+/**
+ * Utility class to provide some test methods existing tests use.
+ */
+public interface TestIO {
```

Comment:
The name looks a bit odd to me. I see that it is only used in tests but I think it should still be named appropriately.